HeartRateMonitor.py

```diff
@@ -0,0 +1,21 @@
+class HRM:
+    avghr = -1
```

Comment:
Unclear what a value of `-1` should represent; would be better to make the code "self-documenting" if possible.